pkg/network/apis/network/validation/validation.go

```diff
@@ -36,9 +37,9 @@ func ValidateClusterNetwork(clusterNet *networkapi.ClusterNetwork) field.ErrorLi
 		} else {
 			maskLen, addrLen := clusterIPNet.Mask.Size()
 			if clusterNet.HostSubnetLength > uint32(addrLen-maskLen) {
-				allErrs = append(allErrs, field.Invalid(field.NewPath("hostSubnetLength"), clusterNet.HostSubnetLength, "subnet length is too large for clusterNetwork"))
```

Comment:
Are you sure this should not be camelCase?

Reply:
Yes (refer https://github.com/openshift/origin/blob/master/pkg/network/apis/network/v1/types.go)
Even though camel case is the general norm, we incorrectly published the field name as 'hostsubnetlength', so this error should refer to the same name.
Anyway this field under ClusterNetwork is deprecated and the new HostSubnetLength under ClusterNetworkEntry is using camel case notation.